subscription.go

```diff
@@ -43,6 +43,7 @@ type Subscription struct {
 	TrialPeriod             bool                 `xml:"trial-period"`
 	Transactions            *Transactions        `xml:"transactions"`
 	Options                 *SubscriptionOptions `xml:"options"`
+	StatusHistory           *StatusHistory       `xml:"status-history"`
```

Comment:
Is it possible to omit the `StatusHistory` type and simply make this `[]*StatusEvent` with a tag `xml:"status-history>status-event"`?

Reply:
I don't think having slices with pointers is a good idea. Why not just []StatusEvent instead? That way there is no need to check a pointer for nil every time you iterate over the slice